Cargo.toml

```diff
@@ -134,7 +134,7 @@ codegen-units = 1
 
 [profile.testnet]
 inherits = "release"
-debug = 1 # debug symbols are useful for profilers
+debug = 2 # debug symbols are useful for profilers
```

Comment:
please revert this

Reply:
Yikes accidentally merged local master :face_in_clouds: - fixed.